hack/jenkins/upload-to-gcs.sh

```diff
@@ -46,8 +46,22 @@ if [[ ! ${JENKINS_UPLOAD_TO_GCS:-y} =~ ^[yY]$ ]]; then
   exit 0
 fi
 
+GCS_SUBDIR=""
+readonly remote_git_repo=$(git config --get remote.remote.url | sed 's:.*github.com/::' || true)
+if [[ -n "${remote_git_repo}" ]]; then
+  case "${remote_git_repo}" in
+    kubernetes/kubernetes) GCS_SUBDIR="" ;;
+    kubernetes/*) GCS_SUBDIR="${remote_git_repo#kubernetes/}/" ;;
```

Comment:
I think eliding it is nice, because it lets things like `k8s-gubernator.appspot.com/pr/test-infra/34` do "the right thing" without extra logic. Especially as we add more and more other repos, having less duplicate `kubernetes_` path-spam will be nice.